copulas/bivariate/base.py

```diff
@@ -220,9 +220,10 @@ def infer(self, X):
     def generator(self, t):
         r"""Compute the generator function for Archimedian copulas.
 
-        The generator is a function :math:`\psi: [0,1]\times\Theta \rightarrow [0, \infty)`
-        that given an Archimedian copula fulills:
+        The generator is a function
+        :math:`\psi: [0,1]\times\Theta \rightarrow [0, \infty)`  # noqa: JS101
```

Comment:
The reason for this `noqa` is that `[0, \infty)` is interpreted as an opening `[` without closing bracket. I'm not sure how to avoid this misinterpretation, so I just `noqa`'d it.